SR2022/2021-11-12-discord-credentials.md

```diff
@@ -0,0 +1,18 @@
+---
+to: Student Robotics 2022 teams
+subject: Student Robotics Discord Details
+---
+
+Hi
+
+This year, we're using [Discord](https://studentrobotics.org/docs/team_admin/discord) as our primary communication method for competitor support. Your invite link is [discord link]().
+
+Upon joining the server users will need to provide a password to gain the correct permissions. For your team, the password is: `password`. Please share both the invite link and the relevant password with your competitors. Discord access should not be shared to those outside your team.
```

Comment:
Is it worth mentioning here that each team has their own channel, so this password is important to ensure that people end up in the right channel?